server/controllers/finance/reports/clientsReport/report.handlebars

```diff
@@ -0,0 +1,55 @@
+{{> head title="REPORT.CLIENTS_REPORT.TITLE" }}
+
+<div class="container-fluid">
+
+  {{> header }}
+
+  <!-- title  -->
+  <h1 class="text-center text-uppercase">
+    {{translate 'REPORT.CLIENTS_REPORT.TITLE'}}
+  </h1>
+
+  <br>
+  <table class="table table-condensed table-striped table-bordered">
+    <thead>
+      <tr class="text-uppercase">
+        <th rowspan="2" class="text-center">{{ translate 'ACCOUNT.NUMBER' }}</th>
+        <th rowspan="2" class="text-center">{{ translate 'FORM.LABELS.DESIGNATION' }}</th>
+        {{#if detailPrevious}}
+          <th colspan="2" class="text-center">{{ translate 'REPORT.CLIENTS_REPORT.PREVIOUS_MVT'}} : {{date fy.openningBalanceDate}}</th>
+          {{else}}
+          <th rowspan="2" class="text-center">{{ translate 'FORM.LABELS.BALANCE' }}<br /> {{date fy.openningBalanceDate}}</th>
+        {{/if}}
+        <th colspan="2" class="text-center">{{ translate 'REPORT.CLIENTS_REPORT.CURRENT_MVT' }}</th>
+        <th rowspan="2" class="text-center">{{ translate 'FORM.LABELS.BALANCE' }}<br /> {{date dateTo}}</th>
+      </tr>
+      <tr class="text-uppercase">
+        {{#if detailPrevious}}
+          <th class="text-center">{{ translate 'FORM.LABELS.DEBIT' }}</th>
+          <th class="text-center">{{ translate 'FORM.LABELS.CREDIT' }}</th>
+        {{/if}}
+        <th class="text-center">{{ translate 'FORM.LABELS.DEBIT' }}</th>
+        <th class="text-center">{{ translate 'FORM.LABELS.CREDIT' }}</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {{#annualLine lines detailPrevious enterprise.currency_id}}
```

Comment:
Can the build in iterator be used instead of these complex helpers? 

Something like:
```html
{{#each lines}}
<tr>
<td>{{this.accountNumber}}<td>
<td>{{this.name}}</td>
{{#if detailPrevious}}
<td>{{this.initDebit}}</td>
<td>{{this.initCredit}}</td>
</tr>
{{/if}}
```